orthoinference/pom.xml

```diff
@@ -23,6 +28,11 @@
 		    <artifactId>json-simple</artifactId>
 		    <version>1.1.1</version>
 		</dependency>
+		<dependency>
+			<groupId>org.jdom</groupId>
```

Comment:
I quickly skimmed the code and didn't see any references to jdom so I'm not sure why it's a dependency here... did i miss it?

Reply:
When I was testing the code outside orthoinference, there was a required jdom dependency in `PredictedPathwayDiagramGeneratorFromDB`